src/server/game/Battlegrounds/BattlegroundMgr.cpp

```diff
@@ -1000,7 +1000,8 @@ void RandomBattlegroundSystem::Update(uint32 diff)
             small.push_back(BATTLEGROUND_AB);
             small.push_back(BATTLEGROUND_SA);
 
-            auto rng = std::default_random_engine{};
+            std::random_device rd;
+            auto rng = std::default_random_engine{rd()};
 
             std::shuffle(big.begin(), big.end(), rng);
```

Comment:
random_shuffle is not better ?

example:
`std::random_shuffle(big.begin(), big.end());`

Reply:
before 3a43e8c415cee4041f2a59e9ebc8af721a49be23#diff-85ba4e80476ac43de064d54592507596  ```random_shuffle``` was used, also  ``` shuffle``` is an improvement over ```random_shuffle```